Q1.py

```diff
@@ -0,0 +1,38 @@
+from collections import Counter
+
+class arithematics(list):
+    lst = []
+    def __init__(self, lst):
+        self.lst = lst
+        self.len = len(lst)
+
+    def mean(self):
+        sum = 0        
+        for i in self.lst:
+            sum += i
```

Comment:
you can use inbuilt `sum` function for this

Reply:
Done. In latest code I used sum inbuilt function. Thanks for noticing.